README.md

```diff
@@ -73,6 +73,12 @@ docstr-coverage some_project/src
 
 #### Options
 
+- _--output=\<type\>, -o \<type\>_ - Set the output target (default stdout)
+  - stdout - Output to standard STDOUT.
```

Comment:
```suggestion
  - stdout-log - Output to standard STDOUT using pythons `logging`.
```

This would give us an option to later adapt/add a print as well. Not sure if we want that, in the end it's @HunterMcGushion 's call. If we do; I'd suggest we use my suggested change (of course not just here in the docs, but everywhere :-))

Reply:
OK. We're going to wait for @HunterMcGushion to decide before pushing?